resume_code/wos_api.py

```
@@ -0,0 +1,288 @@
+    start = datetime.strptime(first_pub_date, "%Y-%m-%d")
+    years = (datetime.now() - start).days / 365.25
+    return round(h_index / years, 1) if years > 0 else 0
+
+
+def load_cache():
+    """Load cached API data if fresh (< 24 hours old)."""
+    if CACHE_FILE.exists():
+        with open(CACHE_FILE) as f:
+            cache = json.load(f)
+        cached_at = datetime.fromisoformat(cache.get("cached_at", "2000-01-01"))
+        if (datetime.now() - cached_at).total_seconds() < 86400:
```

Ruff (0.16.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)